### PR TITLE
Fixed the crush and glitch in multiple output channels

### PR DESCRIPTION
**Pull request by tomoyanonymous**
Continue from #90.

I removed `adjusted_ochannels` in the end and it look working without it.

Could you try if there's no probelm? > @yutannihilation

### CODE REVIEW

**Comment by yutannihilation** [comment]
Thanks, confirmed! This looks simpler and better than mine!

File: mimium-audiodriver/src/backends/cpal.rs [hunk 1]
```diff
@@ -111,15 +98,15 @@ impl NativeAudioData {
                     for i in 0..dspout {
                         o[i] = res[i] as f32;
                     }
-                    for i in dspout..hout {
-                        o[i] = 0.;
-                    }
                 }
                 (1, 2) => {
                     o[0] = res[0] as f32;
                 }
```
Comment (yutannihilation) [inline mimium-audiodriver/src/backends/cpal.rs:104]: This case should probably be covered by the next `(hout, _)` case.

Comment (tomoyanonymous) [reply mimium-audiodriver/src/backends/cpal.rs:104]: Indeed. I was wondering if we should mix stereo signal when the case of mono output hardware && stereo DSP code. For now, I'll simply delete the case above.

Comment (yutannihilation) [reply mimium-audiodriver/src/backends/cpal.rs]: I see. Either should be fine.

**Pull request merged**